src/extension/background-script/actions/ln/getTransactions.ts

```diff
@@ -9,19 +9,22 @@ const getTransactions = async (message: MessageGetTransactions) => {
   const connector = await state.getState().getConnector();
   try {
     const result = await connector.getTransactions();
-
-    let transactions: ConnectorTransaction[] = result.data.transactions
-      .filter((transaction) => transaction.settled)
-      .map((transaction) => {
-        const boostagram = utils.getBoostagramFromInvoiceCustomRecords(
-          transaction.custom_records
-        );
-        return {
-          ...transaction,
-          boostagram,
-          paymentHash: transaction.payment_hash,
-        };
-      });
+    const isNWC = connector.connectorType == "nwc";
```

Comment:
could this settled filtering be done at the connector level for other connectors to remove this conditional?

Reply:
changes in filtering logic are rare but if we wanna change something in future for this part of code we would have to reitrate over all the connectors. so i thought conditional is much more better than spreading filtering across all the connectors

Reply:
moved it to connector level